src/secrets/quote.txt

```diff
@@ -0,0 +1 @@
+There are no secrets better kept than the secrets everybody guesses.
```

Comment:
Is this file being used?  If not it should be removed.

Reply:
Nope, it's not being used. Removed in https://github.com/cisagov/domain-manager-api/pull/194/commits/11c9b5590641f26779aed76b43f4f0140d93bbe1